src/users/Doctor.java

```diff
@@ -1,4 +1,53 @@
 package src.users;
 
-public class Doctor {
+import src.clinic.Clinic;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Doctor extends User{
+
+
+    private List<Integer> expertiseId;
```

Comment:
create `Expertise` class {
private int expertiseId;
private String expertiseName;
}

replace this List with `private HashSet<Expertise> expertises